src/Glpi/Http/ListenersPriority.php

```diff
@@ -34,11 +34,23 @@
 
 namespace Glpi\Http;
 
+use Glpi\Http\Listener\CheckGlpiConfigListener;
 use Glpi\Config\LegacyConfigProviderListener;
+use Glpi\Http\Listener\LegacyAssetsListener;
+use Glpi\Http\Listener\LegacyItemtypeRouteListener;
+use Glpi\Http\Listener\LegacyRouterListener;
+use Glpi\Http\Listener\PluginsRouterListener;
+use Glpi\Http\Listener\RedirectLegacyRouteListener;
+use Glpi\Http\Listener\RootDocListener;
+use Glpi\Http\Listener\SessionStartListener;
 
 final class ListenersPriority
 {
     public const LEGACY_LISTENERS_PRIORITIES = [
+        RootDocListener::class => 700,
+        SessionStartListener::class => 700,
+        CheckGlpiConfigListener::class => 600,
```

Comment:
These 3 listeners should be moved after the `RedirectLegacyRouteListener` listener. Indeed, all previous listeners can be executed first, they do not require the DB to be installed/up-to-date.